shared/components/App/App.jsx

```diff
@@ -4,11 +4,13 @@ import { ThemeProvider } from 'glamorous'
 import themeVariables from '../../styles/theme'
 
 import Homepage from '../../pages/Homepage'
+import ContinentHikes from '../../pages/ContinentHikes/ContinentHikes'
```

Comment:
Remind me to alias our paths so we don't have to keep doing relative paths i.e `../../../../this/is/annoying/right`

Reply:
yeah and also we have to set up the IDE to read the alias.